modules/zivid/camera.py

```diff
@@ -111,6 +113,24 @@ def capture(self, settings_collection=None):
             )
         return Frame(self.__impl.capture())
 
+    def capture_2d(self, settings_2d):
```

Comment:
Camera.capture() works with no arguments, but Camera.capture_2d() does now. Is this intentional?